core-services/image-mirroring/openshift/mapping_origin_4_6

```diff
@@ -7,5 +7,4 @@ registry.ci.openshift.org/origin/4.6:installer-artifacts quay.io/openshift/origi
 registry.ci.openshift.org/origin/4.6:libvirt-installer quay.io/openshift/origin-libvirt-installer:4.6 quay.io/openshift/origin-libvirt-installer:4.6.0
 registry.ci.openshift.org/origin/4.6:machine-os-content quay.io/openshift/origin-machine-os-content:4.6 quay.io/openshift/origin-machine-os-content:4.6.0
 registry.ci.openshift.org/origin/4.6:openstack-installer quay.io/openshift/origin-openstack-installer:4.6 quay.io/openshift/origin-openstack-installer:4.6.0
-registry.ci.openshift.org/origin/4.6:ovirt-installer quay.io/openshift/origin-ovirt-installer:4.6 quay.io/openshift/origin-ovirt-installer:4.6.0
```

Comment:
Is this change necessary?

Reply:
See https://github.com/openshift/release/pull/51536#discussion_r1586851487